roles/device/tasks/install.yml

```diff
@@ -152,7 +153,7 @@
     msg: "{{ tailscale_start.stdout | replace(tailscale_authkey, 'REDACTED') | regex_replace('\\t', '') | split('\n') }}"
   when:
     - tailscale_start is failed
-    - tailscale_start.stdout | length > 0
+    - not ansible_check_mode and tailscale_start.stdout | length > 0
```

Comment:
Each item in this list is an "and" statement on the when condition. Can you add a new item to the list instead of combining on this line?